src/components/blocks/download-button/_download-button.scss

```diff
@@ -0,0 +1,268 @@
+.js-modal-action.b-icon-link:hover .b-icon-link__container::after {
+    opacity: 1;
+}
+
```

Comment:
these overrides aren't good. bad code separation. each component needs to be as independent as possible.

also, haven't said before, but another convention we have is to add classes to html name-spaced with **js-** when they are need by the JS to hook into - they shouldn't appear in the CSS at all. That just means swapping to use the corresponding style classNames, e.g. **b-modal-action** instead of **js-modal-action**

Reply:
moved any modifications to existing elements as modifiers in their respective scss files

Reply:
i agree w/ later point, 'js-' classes arent used in any of the new code anyway it was only so i wouldnt have to reimplement a modal again but i didnt want to change the modal code because it's used on other sites